charge.h

```diff
@@ -0,0 +1,14 @@
+#ifndef CHARGE_H
+#define CHARGE_H
+
```

Comment:
Good practice to wrap header files to avoid multiple inclusion.

Reply:
Thanks a lot Yash